tests/base_test.py

```diff
@@ -75,6 +76,13 @@ def _take_first_new_app_name(self, mozwebqa):
         home_page = Home(mozwebqa)
         home_page.go_to_homepage()
 
-        home_page.click_new_tab()
         app_name = home_page.first_new_app_name
         return app_name
+
+    def _create_new_user(self):
```

Comment:
I'm not sure that we need to prefix these with `_`. They are methods in a `BaseTest` object, so I wouldn't expect anyone to try to call them from elsewhere. I think the `_` is redundant, and it's also not something we usually do in these objects.